sass/components/html/file--link.html

```diff
@@ -1,8 +1,11 @@
 <div class="file file--link">
-  <span class="file__icon icon icon--file-link"></span>
```

Comment:
Well, i would say that the less we change the better is, even though this looks more like an extension than just an "alteration" of the existing modifier we had.
file--link was not showing any "meta" information about the file and now it is, for instance.
Still, if it's not causing headaches i would reintroduce the <span class="icon icon--file file__icon"></span> for consistency.